data/remnant/remnant 1 introduction.txt

```diff
@@ -3741,4 +3741,23 @@ mission "Remnant: Expanded Horizons Storms 1"
 			`	She looks at you quizzically for a moment, and then her display shifts to a mass of symbols. She skims briefly. "I have never even heard of such a unit. Apparently it is a very old one, though. Our records say that it had largely stopped being used by early in the 21st century. Well, I learned something new today." The display and its field of symbols vanishes as she slings her backpack over her shoulder.`
 			label stormend
 			`	"Thanks for the ride, Captain. I have to get working on this data, but if you are interested in helping some more, I hope to have more research expeditions planned soon."`
-			
\ No newline at end of file
+
+
+
+mission "Remnant: Sign Fonts"
+	invisible
+	minor
+	source "Viminal"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+	on offer
+		conversation
+			`You almost never enter a Remnant outfitter from the public entrance, the way you would in human space. You come through the repair bays dockside, past the technicians working on an ibis with an asterioid scanner and a couple of inhibitor cannons stacked next to it. You are walking along side a hover cart with a triple plasma core and some heat shunts that are destined for the refurbishing team. Once receiving has paid you for the salvage, you head past the small break room with the worn seating but very good coffee machine and into the small showroom.`
```

Comment:
```suggestion
			`You almost never enter a Remnant outfitter from the public entrance, the way you would in human space. You come through the repair bays dockside, past the technicians working on an Ibis with an asteroid scanner and a couple of Inhibitor Cannons stacked next to it. You are walking along side a hover cart with a Korath plasma core and some heat shunts that are destined for the refurbishing team. Once receiving has paid you for the salvage, you head past the small break room with the worn seating but very good coffee machine and into the small showroom.`
```
Paid me for the salvage? Why would I rarely enter through the public entrance for Remnant outfitters alone?